### PR TITLE
added requirements file for windows and edited md

### PR DESCRIPTION
**Pull request by Tejas-Choudekar**

### CODE REVIEW

**Review by swapnalisonkusale — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,7 +8,13 @@ Team members:
 - [Swapnali Sonkusale]()
 - [Tejas Choudekar]()
 
-## Virtual Environment
```
Comment (cristian-castro-a) [inline README.md:11]: We should keep this as "Virtual Environment & Dependencies"

File: README.md [hunk 2]
```diff
@@ -8,7 +8,13 @@ Team members:
...
+## Windows
+This project is based on Python 3.9. For windows operating system follow below steps to get the project started:
+1. Install Anaconda
+2. Go to anaconda and write `where conda` it will give you three paths
+3. Add `C:\Users\user_\anaconda3` `C:\Users\user_\anaconda3\Scripts` `C:\Users\user_\anaconda3\Library\bin` path to environment variables in path variable
+4. Install libraries using `conda install --file requirements_windows.txt`
+## MacOs Virtual Environment
```
Comment (cristian-castro-a) [inline README.md]: MacOS

**Review by cristian-castro-a — CHANGES_REQUESTED** [review]
Conda is a manager for virtual environments. Are you creating a virtual environment or only installing in your main manager? It is always recommended to use virtual environments, as each specific project might have own necessary dependencies.

**Comment by Tejas-Choudekar** [comment]
> Conda is a manager for virtual environments. Are you creating a virtual environment or only installing in your main manager? It is always recommended to use virtual environments, as each specific project might have own necessary dependencies.

For windows we usually use Anaconda Prompt for working with jupyter notebook. Since we are using IDE this time, we need these additional steps otherwise Conda is not recognised as a command

File: README.md (hunk 1 above)
Comment (swapnalisonkusale) [reply README.md:11]: Done

File: README.md (hunk 2 above)
Comment (swapnalisonkusale) [reply README.md]: Done

**Review by cristian-castro-a — APPROVED** [review]

**Pull request merged**